src/System.Reflection/tests/MockDerivedType.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Runtime.InteropServices;
+
+namespace System.Reflection.Tests
+{
+    internal class MockDerivedType : Type
```

Comment:
There is `MockType` under Common already. Inherit from that so that you do not have to list overrides from all properties from it?

Once you do that, it should be pretty small. It can be in the same file as the test that it is testing.